tests/unit/test_openapi/conftest.py

```diff
@@ -54,7 +54,7 @@ def get_persons(
         ) -> List[DataclassPerson]:
             return []
 
-        @post(media_type=MediaType.TEXT, sync_to_thread=False)
+        @post("/", media_type=MediaType.TEXT, sync_to_thread=False)
```

Comment:
does this mean the new api forces you to write a path ?

Reply:
No, it's an unrelated change. I just personally find it easier to read like this as it's more explicit. IMO we should remove the option to not pass in a path at all :eyes:

Reply:
Yes we should remove it...to avoid the problem of `@post` instead of `@post()` by the way.

Reply:
I agree with both of you